_includes/navigation.html

```diff
@@ -15,12 +15,12 @@
 
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
         <ul class="navbar-nav ml-auto">
-            <li class="nav-item{% if include.active == "download" %} active{% endif %}">
-                <a class="nav-link py-2 py-lg-4 px-2 px-xl-3" href="{{ site_prefix }}/download">{{ strings.navigation.download }}</a>
+            <li class="nav-item{% if include.active == "github" %} active{% endif %}">
+                <a class="nav-link py-2 py-lg-4 px-2 px-xl-3" href="https://github.com/{{ site.data.contact.github_username }}" target="_blank" rel="noreferrer noopener">GitHub</a>
                 <div class="highlight d-none d-lg-inline-block"></div>
             </li>
-            <li class="nav-item{% if include.active == "blog" %} active{% endif %}">
-                <a class="nav-link py-2 py-lg-4 px-2 px-xl-3" href="{{ site_prefix }}/blog">{{ strings.navigation.blog }}</a>
+            <li class="nav-item{% if include.active == "download" %} active{% endif %}">
+                <a class="nav-link py-2 py-lg-4 px-2 px-xl-3" href="{{ site_prefix }}/">{{ strings.navigation.download }}</a>
```

Comment:
The non GitHub/Download page stuff looks fine, but why are we adding GitHub to the Nav bar and removing the well craft and automated Download pate?

Reply:
Per the request of @roybotbot He wanted to add the GitHub page on the far left and remove the Blog tab. Which moved the download page to the right 1 slot